src/lib/y2storage/proposal/lvm_creator.rb

```diff
@@ -265,6 +266,23 @@ def name_taken?(name, root)
           root.lvm_lvs.any? { |lv| lv.lv_name == name }
         end
       end
+
+      # Returns a list of planned logical volumes adjusting the size
+      #
+      # All logical volumes which sizes are specified as a percentage will get
+      # their minimal and maximal sizes adjusted.
+      #
+      # @param lvs [Array<Planned::LvmLv>] List of planned logical volumes
+      # @param vg  [Planned::LvmVg]        Volume group to place logical volumes on
+      # @return [Array<Planned::LvmLv] New list of planned logical volumes with adjusted sizes
+      def planned_lvs_in_vg(lvs, vg)
```

Comment:
NP: I would clone all lvs or none. I mean, I would expect as result a list of completely new lvs or a list with all the original lvs but some of them modified. Anyway, it can be stay at it is.

Reply:
Actually, it is a bug. Fixed.